sync_radar_data_globus.py

```diff
@@ -157,9 +155,10 @@ def synchronize(self):
         try:
             # You absolutely need the '~' in front of the root of the path for the patterns to work.
             # This isn't obvious from documentation.
-            listing_path = "~/chroot/sddata/{}/{}/{}/".format(self.data_type,
-                                                              self.sync_year,
-                                                              self.sync_month)
+            listing_path = "{}/{}/{}/{}/".format(self.mirror_root_dir,
```

Comment:
Here would be a nice place for that {root}/{sub}/{dir}/{path} :) just makes things more clear

Reply:
Cool. Updated